crypto/keys/secp256k1/secp256k1_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 
 	btcSecp256k1 "github.com/btcsuite/btcd/btcec"
-	"github.com/btcsuite/btcutil/base58"
+	"github.com/cosmos/btcutil/base58"
```

Comment:
Who is going to maintain btcutil? Is it the SDK core team? Some other team?

ref: https://github.com/cosmos/cosmos-sdk/issues/10082#issuecomment-913029326

Reply:
If allowed, I can maintain cosmos/btcutil. It's not updated often and SDK utilizes only part of it.
If SDK core team want to handle this by themselves, it would be good too.

Reply:
I will give anyone admin or write access if they request it. But the SDK team will not maintain it.

Reply:
we should add a bot to downstream any changes, but upstream seems to move slow as well

Reply:
Upstream got 4 news commits yesterday, are they relevant to us?

Reply:
> I will give anyone admin or write access if they request it. But the SDK team will not maintain it.

this should be fine, but if things land in the sdk we shouldnt be giving perms to anyone, there should still be a process for merging things into that repo

Reply:
@amaurym The latest 4 commits are about bech32m encoding (described by [BIP-350](https://github.com/bitcoin/bips/blob/master/bip-0350.mediawiki) instead of original [BIP-173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki)). AFAIK bech32m is not used in SDK yet, but I will merge these commits to cosmos/btcutil for future use.

The forked repo `cosmos/btcutil` contains the changes for #10025. These changes enable bypassing normalization or checksum validation for performance when we are sure that a given address is valid. I'm not sure theses changes will be accepted to the upstream, because it's a micro-optimization and bitcoin friends put the safety first.

@marbar3778 If there is a policy for maintaining repository under `github.com/cosmos`, I will set up the same process and follow the rule when merging changes.